app/src/main/java/com/mapbox/maps/testapp/examples/LocationComponentActivity.kt

```diff
@@ -135,6 +139,16 @@ class LocationComponentActivity : AppCompatActivity() {
         item.isChecked = true
         return true
       }
+      R.id.toggle_opacity -> {
+        val location = binding.mapView.location
+        location.locationPuck = location.locationPuck.run {
+          when (this) {
+            is LocationPuck3D -> copy(modelOpacity = if (modelOpacity == 1.0F) 0.5F else 1.0F)
+            is LocationPuck2D -> copy(opacity = if (opacity == 1.0F) 0.5F else 1.0F)
```

Comment:
Have you checked for values less than `0` and greater than `1`?

Reply:
Yes, the following warnings are visible in logcat:
```
[maps-core]: {ox.maps.testapp}[Style]: Attempt to set an invalid property value 2.000000)
[maps-core]: {ox.maps.testapp}[Style]: Attempt to set an invalid property value -0.500000)
```